addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Hi @neohusky,

Sorry, I meant to reply to your email. Pre-xmas was a little busy! Can I ask why the source is being copied into this project rather than picking it up from maven? If required I can change the license and accept pull requests to the original project. If changing the license my preference would be ASL2, would that be ok?